infra/modules/gcp-tf-runner/read-tfvars.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+variable=$1
+filename="${2:-terraform.tfvars}"
+
+# TODO: extend to look at all *.tfvars files in the directory, not just terraform.tfvars
+
+json="{"
+
+
+# Parse the file, find variable, convert it to JSON
+# this approach of parsing it from a `terr
+# alternative to parsing it ourselves would be `echo var.$variable | terraform console`, but that is
+# slow and brittle to errors/warnings from terraform console.
+if [ -f $filename ]; then
+  while IFS='=' read -r key value; do
+      # Skip commented lines
+      [[ $key =~ ^#.*$ ||  -z $key ]] && continue
```

Comment:
Forgot to tell you that this doesn't work for nested objects, which we luckily don't have

Reply:
yeah; also lists won't work. can add a comment